Feature/weheat integration (#33864)
home-assistant/home-assistant.github.io: source/_integrations/weheat.markdown

---
title: Weheat
description: Instructions on setting up Weheat within Home Assistant.
ha_category:
  - Energy
  - Climate
  - Sensor
ha_iot_class: Cloud Polling
ha_release: "2024.10"
ha_config_flow: true
ha_codeowners:
  - '@jesperraemaekers'
  - '@kjell-van-straaten'
ha_domain: weheat
ha_platforms:
  - sensor
ha_integration_type: integration
---

The **Weheat** {% term integration %} allows you to display your [Weheat](https://www.weheat.nl/) devices through Home Assistant.

> 

Reviewer: ```suggestion

## Prerequisites

- You need a Weheat account before you can use this integration.
- When adding the integration, you will be prompted to enter a **Name**, **Client ID**,  and **Client Secret**. The name is arbitrary, the ID and secret are provided in the [knowledge base](https://support.weheat.nl/s/article/Is-er-een-offici%C3%ABle-Home-Assistant-integratie).
- You need you Weheat **username** and **password**.
 
```

> ## Configuration
> 
> To add the Weheat integration, follow these steps:
> 
> 1. Go to your instance of Home Assistant.
> 2. Navigate to **Settings**.
> 3. Go to **Devices & Services**.
> 4. Click the **+ Add Integration** button.
> 5. Search for **Weheat**.
> 6. You will be prompted to fill in a **Name**, **Client ID**  and **Client Secret**. The name is arbitrary, the ID and secret are provided in the [knowledge base](https://support.weheat.nl/s/article/Is-er-een-offici%C3%ABle-Home-Assistant-integratie).
> 7. You are now redirected to the Weheat login provider. Log in using your **username** and **password**.
> 8. After login, press **link account** to link your account.
> 9. Your device will automatically be added.
> 10. You are all set and ready to go!

Reviewer: ```suggestion
{% include integrations/config_flow.md %}

```

> 10. You are all set and ready to go!

Reviewer: Please add a section called ## Sensors, listing names, units, and descriptions of the sensors.

Reviewer 2: Thanks for the start, we'll fill it in now.

> ## Configuration
> 
> To add the Weheat integration, follow these steps:
> 
> 1. Go to your instance of Home Assistant.
> 2. Navigate to **Settings**.
> 3. Go to **Devices & Services**.
> 4. Click the **+ Add Integration** button.
> 5. Search for **Weheat**.
> 6. You will be prompted to fill in a **Name**, **Client ID**  and **Client Secret**. The name is arbitrary, the ID and secret are provided in the [knowledge base](https://support.weheat.nl/s/article/Is-er-een-offici%C3%ABle-Home-Assistant-integratie).
> 7. You are now redirected to the Weheat login provider. Log in using your **username** and **password**.
> 8. After login, press **link account** to link your account.
> 9. Your device will automatically be added.
> 10. You are all set and ready to go!

Reviewer: ```suggestion
## Prerequisites

- You need a Weheat account, **username**, and **password**
- When adding the integration in Home Assistant, you will be prompted to enter a **Name**, **Client ID**,  and **Client Secret**.
  - The name is arbitrary, the ID and secret are provided in the [knowledge base](https://support.weheat.nl/s/article/Is-er-een-offici%C3%ABle-Home-Assistant-integratie).
- During setup of the integration in Home Assistant, you will be redirected to the Weheat login provider. Log in using your **username** and **password**.
  - After login, select **link account** to link your account.

{% include integrations/config_flow.md %}

## Entities

### Sensors

The Weheat integration provides the following sensors:

- **Output power**:
- **Input power**:
- **COP**:

```

Reviewer: this include config_flow element is s standardized section on adding an integration to Home Assistant
![image](https://github.com/user-attachments/assets/b60defed-9be5-4557-b3ba-8a72c1e79d08)

The elements that need to be available before adding the integration are listed in a Prerequisites section

Reviewer: I tried to add a section describing the sensors. please add a description and the units.

---
title: Weheat
description: Instructions on setting up Weheat within Home Assistant.
ha_category:
  - Energy
  - Climate
  - Sensor
ha_iot_class: Cloud Polling

> ha_release: '2024.7.3'

Reviewer 3: ```suggestion
ha_release: "2024.10"
```

> ha_dhcp: true
> ha_config_flow: true
> ha_quality_scale: silver

Reviewer 3: ```suggestion
ha_config_flow: true
```

ha_codeowners:
  - '@jesperraemaekers'
  - '@kjell-van-straaten'
ha_domain: weheat
ha_platforms:
  - energy
  - climate

>   - sensor

Reviewer 3: ```suggestion
ha_platforms:
  - sensor
```

ha_codeowners:
  - '@jesperraemaekers'
  - '@kjell-van-straaten'
ha_domain: weheat
ha_platforms:
  - sensor
ha_integration_type: integration
---

The **Weheat** {% term integration %} allows you to display your [Weheat](https://www.weheat.nl/) devices through Home Assistant.

## Prerequisites

- You need a Weheat account, **username**, and **password**
- When adding the integration in Home Assistant, you will be prompted to enter a **Name**, **Client ID**,  and **Client Secret**.
  - The name is arbitrary, the ID and secret are provided in the [knowledge base](https://support.weheat.nl/s/article/Is-er-een-offici%C3%ABle-Home-Assistant-integratie).
- During setup of the integration in Home Assistant, you will be redirected to the Weheat login provider. Log in using your **username** and **password**.
  - After login, select **link account** to link your account.

{% include integrations/config_flow.md %}

## Entities

### Sensors

The Weheat integration provides the following sensors:

- **Output power**: Thermal power added to the water in Watts
- **Input power**: Electrical power in Watts
- **COP**: The Coefficient of performance between the above two power measurements

> - **Water inlet temperature"**: The heat pump water inlet temperature in °C

Reviewer: ```suggestion
- **Water inlet temperature**: The heat pump water inlet temperature in °C
```
